safe_transaction_service/history/tasks.py

```diff
@@ -63,7 +63,7 @@ def index_erc20_events_out_of_sync_task(
     block_process_limit: Optional[int] = None,
     block_process_limit_max: Optional[int] = None,
     addresses: Optional[ChecksumAddress] = None,
-    number_of_addresses: Optional[int] = None,
+    number_of_addresses: Optional[int] = 100,
```

Comment:
Is `100` the maximum number of addresses to sync (if not provided)?

Reply:
Yes, we don't want to stress the node and by testing around 100-200 seems like a good value